Template/Tag/CustomHtmlTag.web.js

```diff
@@ -141,11 +144,14 @@
             return element && element.innerHTML && element.innerHTML.toLowerCase().indexOf("<script") !== -1;
         }
 
-        function insertNode(parent, child, append)
+        function insertNode(parent, child, append, previousElement)
         {
             if (append || !parent.firstChild) {
-                parent.appendChild(child);
+                return parent.appendChild(child);
             } else {
+                if (previousElement) {
+                    return previousElement.parentNode.insertBefore(child, previousElement.nextSibling);
+                }
```

Comment:
would we return an element as well when `parent.insertBefore` is executed?

Reply:
I missed this, updated